examples/llava/clip.h

```diff
@@ -36,7 +37,7 @@ struct clip_image_f32_batch {
     size_t size;
 };
 
-CLIP_API struct clip_ctx * clip_model_load    (const char * fname, int verbosity);
+CLIP_API struct clip_ctx * clip_model_load    (const char * fname, int verbosity, std::pair<int, int> load_image_size = {448, 448});
```

Comment:
same with `llava.h` and your `minicpmv_wrapper.h`, since it contains
```
#ifdef __cplusplus
extern "C" {
#endif
```
suggesting that it should be a c header

Reply:
updated